### ISSUE TITLE
Issue submitter error

### ISSUE THREAD

**Issue by NicoLeOca** [pymedusa/SickRage#1050]
Error while to submit an error directly via Medusa

2016-09-07 22:53:05 ERROR    Thread-16 :: [34ff295] Failed doing web ui callback: Traceback (most recent call last):
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/sickbeard/server/web/core/base.py", line 270, in async_call
    result = function(**kwargs)
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/sickbeard/server/web/core/error_logs.py", line 138, in submit_errors
    results = self.issue_submitter.submit_github_issue(CheckVersion())
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/sickbeard/issuesubmitter.py", line 151, in submit_github_issue
    if not sickbeard.DEVELOPER and version_checker.need_update():
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/sickbeard/versionChecker.py", line 338, in need_update
    return self.updater.need_update()
AttributeError: 'NoneType' object has no attribute 'need_update'
Traceback (most recent call last):
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/sickbeard/server/web/core/base.py", line 270, in async_call
    result = function(**kwargs)
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/sickbeard/server/web/core/error_logs.py", line 138, in submit_errors
    results = self.issue_submitter.submit_github_issue(CheckVersion())
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/sickbeard/issuesubmitter.py", line 151, in submit_github_issue
    if not sickbeard.DEVELOPER and version_checker.need_update():
  File "/volume1/@appstore/sickbeard-custom/var/SickBeard/sickbeard/versionChecker.py", line 338, in need_update
    return self.updater.need_update()

full logs:
https://gist.github.com/NicoLeOca/f44554e9ed624400e07af2c34a65ee16

**Comment by fernandog** [comment]
@ratoaq2 any better idea how to fix this?

I would just `if not sickbeard.DEVELOPER and version_checker and version_checker.need_update():`

but maybe need to check `if version_checker` before anything.

**Comment by fernandog** [comment]
@NicoLeOca do you have "Check software updated" disabled?

**Comment by NicoLeOca** [comment]
@fernandog nope

**Comment by fernandog** [comment]
@ratoaq2 so i don't know what's is going on

**Comment by ratoaq2** [comment]
There was a network issue:

```
2016-09-07 20:39:33 WARNING  MAIN :: [] Unable to setup GitHub properly with your github login. Please check your credentials. Error: [Errno -3] Temporary failure in name resolution
2016-09-07 20:39:53 WARNING  MAIN :: [] Unable to setup GitHub properly. GitHub will not be available. Error: [Errno -3] Temporary failure in name resolution
```

That cause the GitHub repository variable (`sickbeard.gh`) to not be initialized, and that's used by versionChecker. 

Thanks for reporting it

**Comment by ratoaq2** [comment]
Fixed by #1059
